api/src/main/java/cz/muni/fi/pa165/dto/CreatePerformanceDTO.java

```diff
@@ -9,16 +10,36 @@
  * @author xtrnkal
  */
 public class CreatePerformanceDTO {
-    private LocalDate startDate;
+    private LocalDateTime startDate;
     private Long showId;
     private Long hallId;
+    private String description;
+    private Float price;
 
-    public LocalDate getStartDate() {
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Float getPrice() {
+        return price;
+    }
+
+    public void setPrice(Float price) {
+        this.price = price;
+    }
+
+    public LocalDateTime getStartDate() {
         return startDate;
     }
 
-    public void setStartDate(LocalDate startDate) {
-        this.startDate = startDate;
+    public void setStartDate(String start) {
```

Comment:
On nekontroluje typ pri volani? Respektive, neni tohle hack? :D

Reply:
byl to hack, opravila jsem zpet na LocalDateTime a pridala binder